15.-Project-3-Elections-Scraper.py

```diff
@@ -0,0 +1,155 @@
+
+import requests
+from bs4 import BeautifulSoup as BS
+import csv
+import sys
+import os
+import unicodedata
+
+URL = 'https://volby.cz/pls/ps2017nss/ps3?xjazyk=CZ'
+payload = {'area':None,
+           'municipality': None}
+
+
+def parse_command_line():
+    while True:
+        if len(sys.argv) != 3:
+            print('Not correct number of arguments (2)')
+            break
+        else:
+            arg = sys.argv[1:]
+            count = 0
+            for k, v in payload.items():
+                payload[k] = arg[count]
+                count += 1
+            return payload
+
+
+def scrape_area():
+    parsed_command_line = parse_command_line()
+    r = requests.get(URL)
+    soup = BS(r.text, 'html.parser')
+    result_town = None
+    for table in soup.find_all('table'):
+        for child in filter(lambda x:x != '\n', table.children):
+            for info in filter(lambda x:x != '\n', child):
+                if info.string == parsed_command_line['area']:
+                    result_town = child
+
+    result_town_list = [i for i in filter(lambda x:x != '\n', result_town)]
+    area_link = result_town_list[-1].find('a').get('href')
+    area_url = "/".join(URL.split("/")[:-1]) + "/" + area_link
+    return area_url
+
+
+def scrape_municipality_url_id_name(scrape_area, parse_command_line):
+    parsed_command_line = parse_command_line
+    r = requests.get(scrape_area)
+    soup = BS(r.text, 'html.parser')
+    result_municipality = None
+    for table in soup.find_all('table'):
+        for child in filter(lambda x: x != '\n', table.children):
+            for info in filter(lambda x: x != '\n', child):
+                if info.string == parsed_command_line['municipality']:
+                    result_municipality = child
+
+    result_municipality_list = [i for i in filter(lambda x: x != '\n', result_municipality)]
+    municipality_id = result_municipality_list[0].string
+    municipality_name = result_municipality_list[1].string
+    municipality_link = result_municipality_list[0].find('a').get('href')
+    municipality_url = "/".join(URL.split("/")[:-1]) + "/" + municipality_link
+
+    return [municipality_url, municipality_id, municipality_name]
```

Comment:
Rather use tuple for immutable data than list. You state your intent more clearly. The collection should only serve as data collection you don't want to append anything to it.